models/public/ctdet_coco_dlav0_384-onnx/ctdet_coco_dlav0_384-onnx.md

```diff
@@ -0,0 +1,113 @@
+# ctdet_coco_dlav0_384-onnx
```

Comment:
I'm not sure it makes sense to use the `-onnx` suffix here. This is still a PyTorch model like any other, we just did the ONNX conversion step ahead of time.